full/Dockerfile

```diff
@@ -209,14 +209,14 @@ RUN bash -lc "cargo install cargo-watch cargo-edit cargo-tree cargo-workspaces"
 LABEL dazzle/layer=tool-docker
 LABEL dazzle/test=tests/tool-docker.yaml
 USER root
-ENV TRIGGER_REBUILD=2
+ENV TRIGGER_REBUILD=3
```

Comment:
Note: Incrementing this variable is not needed when you change other lines within this "Dazzle layer block".